ComfortZoneApp/app/src/main/java/com/example/comfortzone/notification/NotificationUtil.java

```diff
@@ -0,0 +1,37 @@
+package com.example.comfortzone.notification;
+
+import static android.content.Context.ALARM_SERVICE;
+import static com.example.comfortzone.ui.HostActivity.REQUEST_CODE;
+
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+
+import java.util.Calendar;
+import java.util.Date;
+
+public class NotificationUtil {
+
+    public static final int NOTIF_TIME = 10;
+
+    public static void notificationSetup(Context context) {
+        Calendar calendar = Calendar.getInstance();
+        calendar.set(Calendar.HOUR_OF_DAY, NOTIF_TIME);
+        calendar.set(Calendar.MINUTE, 0);
+        calendar.set(Calendar.SECOND, 0);
+
+        if (calendar.getTime().compareTo(new Date()) < 0)
+            calendar.add(Calendar.DAY_OF_MONTH, 1);
+
+        Intent intent = new Intent(context.getApplicationContext(), NotificationReceiver.class);
+        PendingIntent pendingIntent = PendingIntent.getBroadcast(context.getApplicationContext(), REQUEST_CODE, intent, PendingIntent.FLAG_IMMUTABLE);
+        AlarmManager alarmManager = (AlarmManager) context.getSystemService(ALARM_SERVICE);
+
+        if (alarmManager != null) {
+            alarmManager.setRepeating(AlarmManager.RTC, calendar.getTimeInMillis(), AlarmManager.INTERVAL_DAY, pendingIntent);
```

Comment:
hmm, should you check if the alarm is already set? if so we should not set it again?